app/controllers/jwt_controller.rb

```diff
@@ -0,0 +1,15 @@
+class JwtController < ApplicationController
+  def user_jwt
```

Comment:
```suggestion
  def create
```

I would also put the controller under app/controllers/users/jwt_controller